### ISSUE TITLE
failed to connect to jdwp control socket: Connection refused

### ISSUE THREAD

**Issue by twindragon6** [topjohnwu/Magisk#5276]
Device: Samsung SM-G900V (KLTE)
Android version: 11
Magisk version name: ac13ac14
Magisk version code: 23019

Lineage-18.1-20220123-nightly-klte-signed
[boot.zip](https://github.com/topjohnwu/Magisk/files/7938899/boot.zip)
[magisk_patched-23019_7sw5f.zip](https://github.com/topjohnwu/Magisk/files/7938905/magisk_patched-23019_7sw5f.zip)

Rooted Logs:
[magisk_log_2022-01-25T19_54_43.log](https://github.com/topjohnwu/Magisk/files/7938911/magisk_log_2022-01-25T19_54_43.log)
[magisk_log_2022-01-25T19_56_29.log](https://github.com/topjohnwu/Magisk/files/7938912/magisk_log_2022-01-25T19_56_29.log)

Rooted Matlog:
[logs-2022-01-25-19-59-07.zip](https://github.com/topjohnwu/Magisk/files/7938915/logs-2022-01-25-19-59-07.zip)

Non-Rooted Logs:
[magisk_log_2022-01-24T21_05_51.log](https://github.com/topjohnwu/Magisk/files/7938897/magisk_log_2022-01-24T21_05_51.log)

build.board: MSM8974
build.bootloader: G900VVRU2DQL1
build.brand: samsung
build.cpu_abi: armeabi-v7a
build.cpu_abi2: armeabi
build.device: kltevzw
build.display: lineage_klte-userdebug 11 RQ3A.211001.001 10038725
build.fingerprint: samsung/kltexx/klte:6.0.1/MMB29M/G900FXXU1CRH1:user/release-keys
build.hardware: qcom
build.host: lineage-buildkite
build.id: RQ3A.211001.001
build.manufacturer: samsung
build.model: SM-G900V
build.product: kltevzw
build.radio: unknown
build.serial: unknown
build.tags: release-keys
build.time: 1642905566000
build.type: userdebug
build.user: buildkite-agent
version.codename: REL
version.incremental: 10038725
version.release: 11
version.sdk_int: 30

Modems:
20180118_G900V_QL1_BASEBAND-FIRMWARE_MODEM
20180118_G900V_QL1_MODEM
20180118_G900V_QL1_MODEM_NON-HLOS

**Comment by vvb2060** [comment]
Is there any evidence that this was caused by magisk?

**Comment by androidacybot** [comment]
> Is there any evidence that this was caused by magisk?

In my experience it is - I've never seen it without it but a lot with - although I'm not really sure how or why but I don't think it actually harms anything either

**Comment by twindragon6** [comment]
Yes
[logs-2022-01-27-21-28-56.zip](https://github.com/topjohnwu/Magisk/files/7955980/logs-2022-01-27-21-28-56.zip)
[magisk_log_2022-01-27T19_00_47.log](https://github.com/topjohnwu/Magisk/files/7955981/magisk_log_2022-01-27T19_00_47.log)
[magisk_log_2022-01-27T21_29_28.log](https://github.com/topjohnwu/Magisk/files/7955982/magisk_log_2022-01-27T21_29_28.log)
[logs-2022-01-27-19-20-22.zip](https://github.com/topjohnwu/Magisk/files/7955983/logs-2022-01-27-19-20-22.zip)

**Comment by topjohnwu** [comment]
#5307

**Comment by GitStoopID** [comment]
Yo I just googled this log I got:

06-11 06:09:44.156 28585 28598 E z_it.rfanalyze: failed to connect to jdwp control socket: Connection refused


I'm beginning to think Magisk is malware as I have it downloaded BUT not yet installed it! 

Perhaps it is trying to build it's dependencies in hopes I'm gonna tap "download" ...haha. Either way it's concerning to be honest as this is NOT the first time...also small...tiny tiny lil apks where being downloaded as well and I caught that before the Java...json and whatever hit the fan! 

~KiTaMiMe 😒
